mkt/developers/templates/developers/payments/premium.html

```diff
@@ -239,14 +239,20 @@ <h2 id="paid-tab-header">
                           {% endfor %}
                         </tbody>
                       </table>
+                      <p class="hint note rest-of-world">
+                        {% trans %}
+                          Because "Rest of World" is selected, the following
```

Comment:
I like that wording. It explicitly says USD and credit card though. So will go for:

As "Rest of World" is selected your app will be available for purchase in all enabled "Other Regions"

... and link it up.